lib/logger_humio_backend.ex

```diff
@@ -215,11 +229,12 @@ defmodule Logger.Backend.Humio do
     client = Keyword.get(opts, :client, @default_client)
     level = Keyword.get(opts, :level, @default_level)
     metadata = Keyword.get(opts, :metadata, @default_metadata)
-    format = Keyword.get(opts, :format, @default_format) |> Logger.Formatter.compile()
+    formatter = Keyword.get(opts, :formatter, Logger.Formatter)
+    format = Keyword.get(opts, :format, nil) |> formatter.compile()
```

Comment:
nit: p[ipe chains should start with a raw value
```suggestion
    format = opts |> Keyword.get(:format, nil) |> formatter.compile()
```